libs/utils/src/common/utils/validate-log-level.ts

```diff
@@ -0,0 +1,22 @@
+import { LogLevel } from '../../modules/telemetry/enums/log-level.enum';
+import { CustomHelpers } from 'joi';
+import Joi from 'joi';
+
+const validLevels = Object.values(LogLevel);
+
+export function validateLogLevel(
+  value: string,
+  helpers: CustomHelpers
+): Joi.ErrorReport | string {
+  const invalid = value
+    .split(',')
```

Comment:
Why have you decided to use "INFO,WARN" instead of ["INFO", "WARN"] that you could just JSON.parse?